pkg/synthetictests/allowedalerts/matches.go

```diff
@@ -41,53 +42,17 @@ type percentileAllowances struct {
 var defaultAllowances = &percentileAllowances{}
 
 func (d *percentileAllowances) FailAfter(alertName string, jobType platformidentification.JobType) time.Duration {
-	allowed := getClosestPercentilesValues(alertName, jobType)
+	allowed, _, _ := getClosestPercentilesValues(alertName, jobType)
 	return allowed.P99
 }
 
 func (d *percentileAllowances) FlakeAfter(alertName string, jobType platformidentification.JobType) time.Duration {
-	allowed := getClosestPercentilesValues(alertName, jobType)
+	allowed, _, _ := getClosestPercentilesValues(alertName, jobType)
```

Comment:
Should we break out separate tests for these when they have to fallback to a fuzzy match, so we can get a clearer signal on when we're failing with exact matches vs when we're failing on fuzzy? It's getting verbose I know, but it seems worth asking.

Reply:
> Should we break out separate tests for these when they have to fallback to a fuzzy match,

I suggest not doing that because our periodics won't fail like this and the per PR we don't really care about beyond extreme breakage.